Percy/utils/utils.cs

```diff
@@ -0,0 +1,120 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace PercyIO.Appium
+{
+  internal class Utils
+  {
+    public static readonly string[] MyConstants = { "OpenQA.Selenium.Appium.Android.AndroidDriver", "OpenQA.Selenium.Appium.iOS.IOSDriver" };
+    public static Object ReflectionMethodHelper(Object obj, String methodName, params object[] args)
+    {
+      try
+      {
+        Type objectType = obj.GetType();
+        MethodInfo method = objectType.GetMethod(methodName);
+        if (method != null)
+        {
+          return method.Invoke(obj, args);
+        }
+
+        return null;
+      }
+      catch (Exception e)
+      {
+        return null;
+      }
+    }
+
+    public static Object ExecuteScript(Object obj, String script)
+    {
+      Type objectType = obj.GetType();
+      MethodInfo method = objectType.GetMethod("ExecuteScript", new Type[] { typeof(string), typeof(object[]) });
+      if (method != null)
+      {
+        return method.Invoke(obj, new object[] { script, new object[] { null } });
+      }
+      return null;
+    }
+
+    public static Object FindElement(Object obj, String by, String value)
+    {
+      try
+      {
+        Type objectType = obj.GetType();
+        MethodInfo method = objectType.GetMethod("FindElement", new Type[] { typeof(string), typeof(string) });
+        if (method != null)
+        {
+          return method.Invoke(obj, new object[] { by, value });
+        }
+        else
+        {
+          AppPercy.Log($"Driver doesn't have method FindElement by: {by}", "debug");
+        }
+      }
+      catch (Exception)
+      {
+        AppPercy.Log($"Got Error while running FindElement by: {by}", "debug");
+      }
+
+      return null;
+    }
+
+    public static Object? ReflectionPropertyHelper(Object obj, String propertyName)
```

Comment:
```suggestion
    public static T? Property<T>(Object obj, String propertyName)
```
[Optional] Also usually make it generic so that boilerplate is reduced

Reply:
done